redirects.yaml

```
@@ -328,6 +328,18 @@ wwwirishfreemaninfocom:
   sslname: 'irishfreemaninfo.com'
   ca: 'LetsEncrypt'
   disable_event: false
+wikirtapptk:
+  url: 'wiki.rtapp.tk'
+  sslname: 'wiki.recaptime.eu.org'
+  redirect: 'wiki.recaptime.eu.org'
+  ca: "LetsEncrypt"
```

Minor, but for consistency:
```suggestion
  ca: 'LetsEncrypt'
```